projects/store/src/lib/store/utils/base-store/base-store.util.ts

```diff
@@ -37,7 +38,7 @@ export const createBaseStoreAssets = <
 
 	// Iben: Set the initial state
 	const initialState = {
-		data: undefined,
+		data: initialStateValue || undefined,
```

Comment:
Can't you do `data?: initialStateValue`

Reply:
Beide zijn goed tbh, ik heb niet echt een voorkeur persoonlijk